components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,765 @@
+
+## 2. Requirements
+
+The requirements in the [Omega-0 design document](OmegaV0ShallowWater) still apply. Additional design requirements for Omega-1 are:
+
+### Omega will be a hydrostatic, non-Boussinesq ocean model.
+See discussion in introduction. The non-Boussinesq formulation uses the full density throughout resulting in governing equations that conserve mass rather than volume. The substantial change is that the ``thickness'' variable, $h$, is now a mass-thickness, $h = \rho \Delta z$. This is explained in the derivation of the [layered equations below](#layered-equations).
+
+### Omega will use TEOS10 for the equation of state.
+See additional [EOS design document](EOS)
+
+### Omega-1.0 will add new terms for the pressure gradient, vertical mixing, and vertical advection.
+See forthcoming design documents on the pressure gradient, vertical mixing, and vertical advection.
+
+## 3. Continuous Equations
+
+The continuous form of the conservation equations are as follows [Kundu et al. 2024](https://www.amazon.com/Fluid-Mechanics-Pijush-K-Kundu/dp/012405935X), chapter 4, eqns 4.7 and 4.22. This is before any assumptions are made, so this is a compressible, non-hydrostatic, non-Boussinesq fluid. Here all variables are a function of $(x,y,z)$, ${\bf u}_{3D}$ denotes the three-dimensional velocity vector, ${\bf u}_{3D} \otimes {\bf u}_{3D} = {\bf u}_{3D}{\bf u}_{3D}^T$ is the tensor product, $\nabla_{3D}$ is the three-dimensional gradient, $D/Dt$ is the material derivative, and other variables defined in the [Variable Definition Section](#variable-definitions) below.
+
+momentum:
+
+$$
+\frac{D \rho {\bf u}_{3D} }{D t} \equiv
+\frac{\partial \rho {\bf u}_{3D}}{\partial t}
+ + \nabla_{3D} \cdot \left( \rho {\bf u}_{3D} \otimes {\bf u}_{3D}  \right)
+  = - \nabla_{3D} p
++ \rho {\bf D}^u + \rho {\bf F}^u
+$$ (continuous-momentum)
+
+mass:
+
+$$
+\frac{D \rho}{D t} \equiv
+\frac{\partial \rho }{\partial t}
+ + \nabla_{3D} \cdot \left( \rho  {\bf u}_{3D}  \right)
+= 0
+$$ (continuous-mass)
+
+tracers:
+
+$$
+\frac{D \rho \varphi }{D t} \equiv
+\frac{\partial \rho \varphi}{\partial t}
+ + \nabla_{3D} \cdot \left( \rho \varphi {\bf u}_{3D}  \right)
+= D^\varphi + Q^\varphi
+$$ (continuous-tracer)
+
+Here we have express the following terms as a general operators, with examples of specific forms provided below: the dissipation ${\bf D}^u$, momentum forcing ${\bf F}^u$, tracer diffusion $D^\varphi$, and tracer sources and sinks $Q^\varphi$. The momentum equation can be rewritten using the product rule on $\rho {\bf u}$, mass conservation, and dividing by $\rho$, as:
+
+$$
+\frac{D {\bf u}_{3D} }{D t} \equiv
+\frac{\partial {\bf u}_{3D}}{\partial t}
++ {\bf u}_{3D}\cdot \nabla_{3D} {\bf u}_{3D}
```

Comment:
When `\rho` varies with space, this is only one of 3 terms that should be here:
$$
∇⋅(ρu⊗u)=ρ(u⋅∇)u+(u⋅∇ρ)u+u(∇⋅(ρu))
$$

Reply:
I follow this now.  Sorry for the noise here!